zdm/tests/test_with_craco.py

```diff
@@ -0,0 +1,154 @@
+""" Run tests with CRACO FRBs """
```

Comment:
Aha! So this is the routine you have used to generate your 1D plots? Great! Maybe we can upload this, I'll upgrade to this version, and I'll first try to reproduce your strange H0 plot, then I'll try to debug those jumps

Reply:
yes, that's right.  And I have moved it into the `craco/` folder and renamed it `testing.py`